### PR TITLE
NEW - Add API to get all repos under a bitbucket project

### PR DESCRIPTION
**Pull request by CCardosoDev**

### CODE REVIEW

File: bitbucket/repos.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+from bitbucket.exceptions import ReposNotFound
+from bitbucket.service import BitbucketServiceV1_0
+from dataclasses import dataclass
+
+log = logging.getLogger(__name__)
+
+
+@dataclass
+class BitbucketReposV1_0(BitbucketServiceV1_0):
```
Comment (rodgomes) [inline bitbucket/repos.py]: minor: shouldn't we abstract the bitbucket version currently in use and keep the class name as `BitbucketRepos`?

File: bitbucket/views.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+from rest_framework.views import APIView
+
+from . import serializers
+from .repos import BitbucketReposV1_0
+
+
+class BitbucketReposView(APIView):
+    serializer_class = serializers.BitbucketReposV1_0Request
+
+    # TODO: add permissions and auth classes
```
Comment (rodgomes) [inline bitbucket/views.py:12]: question: the check that a user can have access to a given project is going to be done as part of the `TODO: add permissions and auth classes`, right?

File: bitbucket/repos.py (hunk 1 above)
Comment (CCardosoDev) [reply bitbucket/repos.py]: Ok, I have changed it, now I don't make reference to the version number.

File: bitbucket/views.py (hunk 2 above)
Comment (CCardosoDev) [reply bitbucket/views.py:12]: As discussed offline, this part should be part of a different story.

**Review by rodgomes — APPROVED** [review]

**Pull request merged**